extensions/amp-story/0.1/amp-story.js

```diff
@@ -269,8 +278,26 @@ export class AmpStory extends AMP.BaseElement {
 
 
   /** @private */
+  isAmpStoryEnabled_() {
+    if (isExperimentOn(this.win, TAG) || getMode().test || getMode().localDev) {
+      return true;
+    }
+
+    const origin = getSourceOrigin(this.win);
+    const hostName = parseUrl(origin).hostname;
+    const domains = hostName.split('.');
+
+    return domains.some((unusedDomain, index) => {
+      const domain = domains.slice(0, index + 1).join('.');
```

Comment:
I'm not sure I understand this. Why not use `unusedDomain` instead of a 1-element `slice()`? 

Can you work through an example in a comment? E.g.

```js
// For a source origin 'https://sub.domain.com', hostName == 'sub.domain.com'
// and domains == ['sub', 'domain', 'com']. 
```